resqpy/olio/vdb.py

```diff
@@ -221,8 +221,8 @@ def __init__(self, fp, place, item_type, count, max_count):
             self.a = np.empty((count, 2), dtype = int)
             self.a[:, :] = nda[:, :]
         else:
-            self.a = np.empty((count,), dtype = dtype)
-            self.a.data = fp.read(count * byte_size)  # todo: check C ordering; endianess etc.
+            b = fp.read(count * byte_size)
+            self.a = np.frombuffer(buffer = b, dtype = dtype, count = count)
```

Comment:
Does frombuffer() have an explicit byte size per element, or is it deduced from count (and the size of buffer), or does dtype determine the size per element?  Vdb is using 32bit floats and ints.

Reply:
It is deduced from count [https://numpy.org/doc/stable/reference/generated/numpy.frombuffer.html](url)